### PR TITLE
Introducing public_partitions_already_filtered

### PR DESCRIPTION
**Pull request by dvadym**

### CODE REVIEW

**Review by monsieurmuffin — COMMENTED** [review]
Thanks! Please could you add a description of the PR adding a bit more details on what the flag is doing and why we need it?

File: pipeline_dp/aggregate_params.py [hunk 1]
```diff
@@ -131,6 +131,9 @@ class AggregateParams:
          max_contributions_per_partition. This option can be used if the dataset
          does not contain any identifiers that can be used to enforce
          contribution bounds automatically.
+        partitions_already_filtered: if true, it is assumed that partitions are
```
[the remark this answers is not in the document]
Comment (dvadym) [reply pipeline_dp/aggregate_params.py]: Thanks for comments! I've addressed them. PTAL

File: examples/movie_view_ratings/run_all_frameworks.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def calc_dp_rating_metrics(movie_views, backend, public_partitions):
         max_contributions_per_partition=1,
         min_value=1,
         max_value=5,
+        partitions_already_filtered=True,
```
Comment (monsieurmuffin) [inline examples/movie_view_ratings/run_all_frameworks.py:86]: nit: should this be public_partitions_already_filtered=True ?

File: pipeline_dp/aggregate_params.py [hunk 3]
```diff
@@ -131,6 +131,9 @@ class AggregateParams:
          max_contributions_per_partition. This option can be used if the dataset
          does not contain any identifiers that can be used to enforce
          contribution bounds automatically.
+        public_partitions_already_filtered: if true, it is assumed that records
+         with partition_key which are not in public_partitions are already
+         removed from the dataset. It applies only for public partitions.
```
Comment (monsieurmuffin) [inline pipeline_dp/aggregate_params.py:136]: "It applies only for public partitions." -> "The flag can only be used with public partitions"

**Review by monsieurmuffin — APPROVED** [review]

File: pipeline_dp/aggregate_params.py (hunk 3 above)
Comment (dvadym) [reply pipeline_dp/aggregate_params.py]: Thanks

**Review by dvadym — COMMENTED** [review]
Thanks for review!

**Pull request merged**